src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs

```diff
@@ -368,7 +368,12 @@ private List<string> AddArgs()
             {
                 foreach (var arg in this.VSTestCollect)
                 {
-                    if (arg.Equals("Code Coverage", StringComparison.OrdinalIgnoreCase))
+                    // Argument value could be CodeCoverage;a=b;c=d.
+                    // Split arg with ; and compare with the first token.
+                    var tokens = arg.Split(';');
+
+                    if (arg.Equals("Code Coverage", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
Maybe better to have const string with value = "Code Coverage" for correctness. If someone changes the value in future we will not get misspelling.

Reply:
Updated